packages/core/src/node/build.ts

```diff
@@ -46,7 +46,7 @@ export async function bundle(
         initRsbuild(docDirectory, config, pluginDriver, true, {
           output: {
             distPath: {
-              server: 'ssr',
+              root: 'doc_build/ssr',
```

Comment:
should use `outDir` or const `OUTPUT_DIR`?

Reply:
yes

Reply:
fixed